week1/homework/src/server.js

```diff
@@ -9,12 +9,36 @@ function createServer(port) {
   let state = 10;
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+
+    function getResponse(state, body) {
+      response.writeHead(state, { 'Content-Type': 'application/json' });
+      response.end(JSON.stringify(body));
+    }
+
+    switch (request.url) {
+      case '/state':
+        getResponse(200, { state });
```

Comment:
oh my god this is awesome Karam!!! 💯 
very very very well done, you should be proud of yourself! 👑